packages/smooth_app/lib/themes/smooth_theme.dart

```diff
@@ -27,14 +38,27 @@ class SmoothTheme {
     COLOR_TAG_BROWN: Colors.brown,
   };
 
+  static Color? getColor(
```

Comment:
`static` or not?

Reply:
I kept this as static because it was used in so many places, and it made it less verbose. It can still be tested, because it uses the instance to provide the data.  I'm happy to replace calls to it with calls to SmoothTheme.instance.getColor instead if you think it's confusing (which I could see it being).